xla/stream_executor/cuda/cuda_dnn.cc

```diff
@@ -4578,6 +4582,322 @@ void FixDimsForRaggedOffset(std::vector<int64_t>& dims, int max_reg_per_batch) {
   dims[0] *= max_reg_per_batch;
 }
 
+class ScoreModFunc {
+  using Tensor_t = std::shared_ptr<cudnn_frontend::graph::Tensor_attributes>;
+  using Graph_t = std::shared_ptr<cudnn_frontend::graph::Graph>;
```

Comment:
Nit: The Google style guide prefers not to have `_t` suffixes. Can you rename these aliases to `Tensor` and `Graph`?